### DISCUSSION TITLE
Install HA-addon

### DISCUSSION THREAD

**Discussion by billerby** [wmbusmeters/wmbusmeters#802] [Q&A]
I am running Home assistant operating system on a dedicated machine and are failing to install this addon for some reason.

settings -> addons -> add-on store -> repositories:

![image](https://user-images.githubusercontent.com/123237/212468908-08ae294d-dd2e-48c3-8e63-ce8bd302722f.png)

So far so good.

But after doing this the addon is nowhere to be found when searching the add-ons. Tried rebooting without success.

Logs are emtpy regarding this.

Any ideas what I am missing?

/Erik

**Comment by pepsonEL** [comment]
I also have the same problem.

**Reply by billerby** [reply]
I can add that I am on the latest version 2023.1.4 and supervisor 2022.12.1 and operating system 9.4

**Reply by pepsonEL** [reply]
i also has the same: 
Home Assistant 2023.1.4
Supervisor 2022.12.1
Operating System 9.4

Repository i also can add but not found any addon in this repo in HA.

**Comment by billerby** [comment]
Managed to install with the Android app.

Den lör 14 jan. 2023 15:34pepsonEL ***@***.***> skrev:

**Reply by pepsonEL** [reply]
???

**Comment by BIBOLV** [comment]
Refresh addon store, navigate away from addon page and after a minute go back and search. If it does not work - restart HA. Helped for me.

**Comment by thecem** [comment]
You have to clear the browser cache, or simply refresh the page with F5 or CTRL + F5. Then you can search your add-on via:

![5EA82470-93CC-45FC-A333-9898FC379AF7](https://user-images.githubusercontent.com/46648579/212529325-585ff904-82d4-4142-a5f1-a96c239656a4.jpeg)

and:

![F60530A9-9B13-44C6-9D03-0550F5E4FB08](https://user-images.githubusercontent.com/46648579/212529353-38bf2846-a701-46cb-86d6-9903803a2428.jpeg)

Select, install (!!!!wait up to 40 min!!!!), configure !
Done
